src/styles/components/containers/_fold.scss

```diff
@@ -158,6 +158,12 @@
       border-width: 1px 1px 1px 1px;
     }
   }
+
+  // if the panel is within a fold, allow overflow for dropdown, etc.
+  .panel {
```

Comment:
please update the existing styles rather than moving them here... rationale: this is styling Panel, so belongs in `_panel.scss`